docs/20_smart-contracts/10_getting-started/20_accounts/index.md

```diff
@@ -0,0 +1,208 @@
+
+
+## Account names
+
+Every account on the EOS Network has a human-readable name. This makes it easier to identify transaction recipients and smart contracts. In order to keep account names efficient on the blockchain, a few restrictions apply to all names: 
+
+
+
+* All characters must be lowercase
+* Every name must be 12 characters long (or less with a suffix/premium name)
+* Only letters `a-z`, numbers `1-5`, and period (`.`) are supported characters. 
+* Names cannot start with a number or a period. 
+* Names cannot end with a period.
+
+Periods have a special meaning for EOS accounts. They specify that an account has a **suffix** (similar to a top-level domain like .com), also known as a **premium name**. Accounts with a suffix can only be created by the **suffix owner**. 
+
+For instance, if someone owns the suffix `.bar` then only that person can create `foo.bar`. 
+
+At the end of this guide, we will talk about the Name Bidding system which allows someone on the EOS network to purchase premium names. 
+
+
+## Public/Private Keys
+
+Every EOS account is ultimately controlled by a public/private key pair. The private key is used to sign transactions and must be kept confidential, while the public key is used to identify the account on the blockchain and can be publicly known.
+
+It is important to safeguard your private key, as it provides complete control over your account's digital assets. Your private key should be stored in a secure software or hardware wallet, as whoever possesses it will have full access to the assets linked to your account.
+
+EOS accounts offer extra security mechanisms out of the box, using something called the *permission system*.
+
+Examples of private and public keys: 
+```
+Private: 5KSdyAiFzYQAtBKDBKCCF28KMMhZ4EmXUxSg8B3nSkHKutT15rY
+Public: PUB_K1_5d7eRKgCCiEdsbBdxxnZdFWnGYS64uWZPZgTcTU1xnB2aESxqR
+Legacy Public Format: EOS5d7eRKgCCiEdsbBdxxnZdFWnGYS64uWZPZgTcTU1xnB2cq4JMD
+```
+
+
+## Permissions system
+
+Each account has a set of hierarchical permissions that control it, and comes with two base permissions by default. These two permissions cannot be removed as they are required for your account to function properly. 
+
+The mandatory permissions are `owner` and `active`.
+
+A permission can only ever change its own information or the information of its children. It can never change its parent’s information.
+
+![Who can change permissions?](images/who_can_change_permissions.png)
+
+
+The information that controls a permission is either another `account@permission` or a public key. This allows for the creation of complex account control structures, where multiple parties can have control over a single account while having full autonomy over their own account’s security. 

```

Comment:
I don't follow this `The information that controls a permission is either another `account@permission` or a public key.` how can a public key control the settings/definition of a permission?

Reply:
By "information" I think it refers to the authority table associated with the permission, which is what controls what the account can do if the threshold is met. This information includes either public keys, other permissions, or waits - https://docs.eosnetwork.com/docs/latest/protocol/accounts_and_permissions#authority-schema

Reply:
I'm guessing you're getting at the fact that technically the private key controls the account and its permissions?

This is true, but the private key isn't actually registered on chain. So as far as the chain is concerned, the public key controls it. But, I can change this to display both since both are correct.